mod/reportedcontent/start.php

```diff
@@ -84,6 +84,7 @@ function reportedcontent_user_hover_menu($hook, $type, $return, $params) {
 	$return[] = \ElggMenuItem::factory([
 		'name' => 'reportuser',
 		'text' => elgg_echo('reportedcontent:user'),
+		'icon' => 'report-this',
```

Comment:
Let's not user Elgg icons anymore. Phase them out. I would push this to the very end of the list. Bullhorn perhaps?